examples/data_view/column_example.py

```diff
@@ -114,7 +101,10 @@ def get_image(self, model, row, column):
                 HasTraitsRowInfo(
                     title="Country",
                     value="address.country",
-                    value_type=CountryValue(flags=flags),
+                    value_type=EnumValue(
+                        values=['Canada', 'UK', 'USA'],
```

Comment:
nitpicky comment - you could use `list(flags.keys())` here instead of hardcoding the list

Reply:
This is possibly better:
```suggestion
                        values=sorted(flags.keys()),
```